### PR TITLE
CONTRIBUTING.md: Add line about commit messages

### PR DESCRIPTION
**Pull request by cdayjr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [N/A] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [N/A] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [N/A] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [N/A] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I saw there was a bot check for the commit messages after I had already went through the checklist above and got my PR ready, I figure it'd make sense to add a line about the commit styles in the CONTRIBUTING.md guide.

### CODE REVIEW

**Review by dawidd6 — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct).
 
+Ensure your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit).
```
Comment (carlocab) [inline CONTRIBUTING.md:5]:
```suggestion
Ensure your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit). Please put the name of the formula first, and modify only one formula per commit.
```

**Review by carlocab — COMMENTED** [review]
Thanks, @cdayjr. One small suggestion.

**Review by carlocab — DISMISSED** [review]
Thanks @cdayjr!

**Comment by carlocab** [comment]
Will leave this open for a bit longer to get more eyes on it.

**Review by nandahkrishna — DISMISSED** [review]
This looks good to me. Although there are some instructions right at the bottom (the point with `run git commit ...`), having this at the top with a link to the Formula Cookbook section would definitely make it more noticeable and should hopefully translate into more contributors adhering to the guidelines.

Just a small suggestion (I don't feel too strongly about this): We may want to prefix the first sentence with `If you are creating a pull request, `. I say this because right below the message, we have the `To report a bug` section, and that doesn't require committing.

**Review by nandahkrishna — DISMISSED** [review]
Thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct).
 
+If you are creating a pull request, ensure your commits follow the
```
Comment (MikeMcQuaid) [inline CONTRIBUTING.md:5]: This is mentioned in "To add a new formula" below and `brew bump-formula-pr` will handle it for you too. Thoughts?

Comment (carlocab) [reply CONTRIBUTING.md:5]: I still pull out my "please use homebrew/core style in your commit messages" saved reply at least once a day, so I think a lot of users still miss it.

I'm fine with making this less verbose (i.e. back to @cdayjr's original suggestion 😅) so easier to scan.

**Comment by MikeMcQuaid** [comment]
Cool, I don't feel strongly, I'll leave to another maintainer to merge this.

**Comment by carlocab** [comment]
@cdayjr I think I'm happy to go back to your original suggestion to keep things less verbose. Verbosity discourages contributors. Or at least, discourages contributors from reading the guidelines. Sorry for flip-flopping on this!

Thoughts, @nandahkrishna?

**Comment by nandahkrishna** [comment]
I'm fine with the original one as well.

**Comment by cdayjr** [comment]
Thanks all. I have removed the additional commits from this branch.

**Review by carlocab — APPROVED** [review]
Thanks, @cdayjr!

**Pull request merged**